src/main/java/com/brandwatch/kafka_pod_autoscaler/triggers/KafkaLagTriggerProcessor.java

```diff
@@ -0,0 +1,50 @@
+package com.brandwatch.kafka_pod_autoscaler.triggers;
+
+import static java.util.Objects.requireNonNull;
+
+import com.google.auto.service.AutoService;
+
+import brandwatch.com.v1alpha1.KafkaPodAutoscaler;
+import brandwatch.com.v1alpha1.kafkapodautoscalerspec.Triggers;
+import io.fabric8.kubernetes.client.KubernetesClient;
+import lombok.extern.slf4j.Slf4j;
+
+import com.brandwatch.kafka_pod_autoscaler.ScaledResource;
+import com.brandwatch.kafka_pod_autoscaler.cache.KafkaMetadataCache;
+
+@Slf4j
+@AutoService(TriggerProcessor.class)
+public class KafkaLagTriggerProcessor implements TriggerProcessor {
+    @Override
+    public String getType() {
+        return "kafka";
```

Comment:
nabd, should this be changed to `kafkaLag`

Reply:
Yeah, that's a breaking change though and I wanted to avoid that

I'll probably change the CRD soon, which would all be one big breaking change

Reply:
The groundwork for that is here: https://github.com/BrandwatchLtd/kafka-pod-autoscaler/pull/67

(as it's easier to change the CRD by refactoring the code to match what we want, instead of trying to morph the CRD into generating the code we want)